pages/1.12/installing/production/system-requirements/index.md

```diff
@@ -218,23 +218,47 @@ sudo yum install -y tar xz unzip curl ipset
 
 ### Cluster permissions (production installation)
 
-On each of your cluster nodes, use the following command to:
+On each of your cluster nodes, follow the following instructions:
```

Comment:
@mhrabovcin:
Replace "On each of your cluster nodes, follow the following instructions:" 
with "On each of your cluster nodes, follow the below instructions: